lib/PuppeteerSharp/Puppeteer.cs

```diff
@@ -1,13 +1,18 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
 
 namespace PuppeteerSharp
 {
     public class Puppeteer
     {
         public static string[] DefaultArgs => Launcher.DefaultArgs;
 
+        public static string GetExecutablePath()
+        {
+            var downloader = Downloader.CreateDefault();
+            var revisionInfo = downloader.RevisionInfo(Downloader.CurrentPlatform, Downloader.DefaultRevision);
+            return revisionInfo.ExecutablePath;
```

Comment:
This code is in the Launcher class on Puppeteer. Let's honor that.

https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/lib/Launcher.js#L188

Reply:
should that method be static?

Reply:
Yes. Let's follow Puppetter API, it's static there.